packages/loader/src/ktx2/KTX2Loader.ts

```diff
@@ -43,11 +43,23 @@ export class KTX2Loader extends Loader<Texture2D | TextureCube> {
     ]
   };
   private static _supportedMap = {
-    [KTX2TargetFormat.ASTC]: [GLCapabilityType.astc],
-    [KTX2TargetFormat.ETC]: [GLCapabilityType.etc],
-    [KTX2TargetFormat.BC7]: [GLCapabilityType.bptc],
-    [KTX2TargetFormat.BC1_BC3]: [GLCapabilityType.s3tc],
-    [KTX2TargetFormat.PVRTC]: [GLCapabilityType.pvrtc, GLCapabilityType.pvrtc_webkit]
+    [KTX2TargetFormat.ASTC]: {
```

Comment:
_supportedMap is an ambiguous name, change it is better

Reply:
change to  '_capabilityMap'